code/datums/skills/_skill.dm

```diff
@@ -4,3 +4,47 @@
 
 /datum/skill/proc/get_skill_speed_modifier(level)
 	return
+
+//Mining
+/datum/skill/mining
```

Comment:
I'd prefer we have just this small definition file than start on a new monolithic file when every new skill gets added in here. How it was before was good.